test/imagekit/spec_helper.rb

```diff
@@ -1,8 +1,8 @@
-require 'simplecov'
-SimpleCov.start 'rails'
+# require 'simplecov'
+# SimpleCov.start 'rails'
```

Comment:
Let's retain the coverage code, but remove just codecov for now

Reply:
done